src/web/app/pages-help/instructor-help-page/instructor-help-courses-section/instructor-help-courses-section.component.ts

```diff
@@ -14,11 +14,48 @@ export class InstructorHelpCoursesSectionComponent extends InstructorHelpSection
 
   readonly supportEmail: string = environment.supportEmail;
 
+  isAddStudentsCollapsed: boolean = false;
+  isSizeLimitCollapsed: boolean = false;
+  isNoTeamsCollapsed: boolean = false;
+  isSectionsCollapsed: boolean = false;
+  isEnrollSectionsCollapsed: boolean = false;
+  isAddInstructorCollapsed: boolean = false;
+  isEditInstructorCollapsed: boolean = false;
+  isInstructorAccessCollapsed: boolean = false;
+  isPrivilegesCollapsed: boolean = false;
+  isViewStudentsCollapsed: boolean = false;
+  isChangeSectionCollapsed: boolean = false;
+  isDisappearedCourseCollapsed: boolean = false;
+  isDelStudentsCollapsed: boolean = false;
+  isArchiveCourseCollapsed: boolean = false;
+  isViewArchivedCollapsed: boolean = false;
+  isCourseUnarchiveCollapsed: boolean = false;
+  isViewDelCollapsed: boolean = false;
+  isRestoreCollapsed: boolean = false;
+  isDelCollapsed: boolean = false;
+  isRestoreAllCollapsed: boolean = false;
+  @Output() collapseStudentEditDetails: EventEmitter<Boolean> = new EventEmitter<Boolean>();
+
   constructor() {
     super();
   }
 
   ngOnInit(): void {
+  }
 
+  /**
+   * To scroll to a specific HTML id
+   */
+  jumpTo(target: string): boolean {
```

Comment:
I think there is a `pageScrollService` integrated in the code base. Can you try to integrate that?

Reply:
Sure, done. Please see the last commits below.